app/src/main/java/com/mashup/mobalmobal/util/NotificationUtils.kt

```diff
@@ -0,0 +1,53 @@
+package com.mashup.mobalmobal.util
+
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import android.os.Build
+import androidx.core.app.NotificationCompat
+import com.mashup.mobalmobal.R
+import com.mashup.mobalmobal.ui.MainActivity
+
+object NotificationUtils {
+    private const val NOTIFICATION_ID = 0
+
+    fun NotificationManager.sendNotification(messageBody: String, applicationContext: Context) {
```

Comment:
```suggestion
    fun NotificationManager.sendNotification(context: Context, messageBody: String) {
```
스타일 차이긴한데 저는 항상 context가 첫번째에 있는게 좋더라구요.
중요도에 따라서 파라미터 순서 바꿔주시면 좋습니다!